### ISSUE TITLE
Installer does not create Custom_Words.txt 

### ISSUE THREAD

**Issue by CedSha** [bharadwaj-raju/TextSuggest#49]
I installed on Ubuntu 16.04 via the download of textsuggest-git.deb. 
After install I got an error message for the file ~/.config/textsuggest/Custom_Words.txt does not exist. (Even launch textsuggest with --help option will give this error). 
After creating manually the file but empty I got some error about json. Only after populating the file got no more error...
The extension is still not working smoothly for me and I'll fill another issue after more investigations.

**Comment by bharadwaj-raju** [comment]
@CedSha Should work now.

What are the things that don't work smoothly?

**Comment by CedSha** [comment]
@bharadwaj-raju I try to install again but this time from the repo and install.sh.
The directories are created but the files still not. Other now works very well :) and this is great extension to my system. I have a pair of suggestions but I believe it maybe difficult to do :
When auto-completing a custom word the list show the abbreviate form I feel it would be great if it could show the completed format. 
When the word did not exist in dictionary, if typing again the word at the top of list should record it in custom dict.
Good luck
Tks
